blog/_posts/2019-08-28-release-process.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: post
+title:  Julia's Release Process
+author: [Stefan Karpinski](https://karpinski.org/) ([Julia Computing](https://juliacomputing.com/))
+---
+
+People involved in the day-to-day development of a project tend to become so familiar with its rhythm and process that they internalize it and it feels like everyone must just _know_ how each stage unfolds. Of course, from the outside looking in it's not so obvious. So I thought it might be helpful to the broader Julia community—and maybe even for other programming language communities—to actually write down Julia's release process, including the details of:
+
+- what kinds of releases there are
+- what can and cannot change in each kind of release
+- what the stages of the release process are
+- who should use which releases based on risk tolerance
+- the phases of a release and sequence of events surrounding it.
+
+This information is collected from a small set of posts on [discourse](https://discourse.julialang.org/) and conversations on [Slack](https://slackinvite.julialang.org/), so the information exists "out there", but this blog post brings it all together in a single place. We may turn this post into an official document if it's well received. Julia follows "semantic versioning" as specified in the [SemVer](https://semver.org/) standard, but SemVer leaves a fair amount of room for interpretation and says very little about process, so this post aims to fill in those details.
+
+# Patch releases
+
+- Patch releases increment the last digit of Julia's version number, e.g. going from `1.2.3` to `1.2.4`.
+
+- Patch releases, following [SemVer](https://semver.org/), should only contain bug fixes, low-risk performance improvements, and documentation updates. Of course, what exactly constitutes a bug fix can be more subjective than one might naïvely imagine since people write code that relies on buggy behavior. In general, we try to be very conservative with patch releases and use [PkgEval](https://github.com/JuliaComputing/NewPkgEvaljl) to ensure that there's minimal risk. People should be confident that they can just upgrade to the latest patch release without worrying about it breaking things.
```

Comment:
The address of PkgEval is missing a dot:
```suggestion
- Patch releases, following [SemVer](https://semver.org/), should only contain bug fixes, low-risk performance improvements, and documentation updates. Of course, what exactly constitutes a bug fix can be more subjective than one might naïvely imagine since people write code that relies on buggy behavior. In general, we try to be very conservative with patch releases and use [PkgEval](https://github.com/JuliaComputing/NewPkgEval.jl) to ensure that there's minimal risk. People should be confident that they can just upgrade to the latest patch release without worrying about it breaking things.
```